test/kitchen/test/integration/common/rspec/spec_helper.rb

```diff
@@ -237,22 +237,6 @@ def is_port_bound(port)
     expect(File).to exist(conf_path)
   end
 
-  it 'has running checks' do
```

Comment:
I think this test is still useful, but we should wait for ~15 seconds for the first check runs to be scheduled.